### PR TITLE
Refactor files.js into FilesController.

### PR DESCRIPTION
**Pull request by nlutsenko**
Nicer structure, get rid of globals, follow-up PR from #315.

### CODE REVIEW

**Comment by maysale01** [comment]
While you're at it, any opposition to moving all the handlers or controllers into a sub directory, starting with this one?

**Comment by nlutsenko** [comment]
Yup, no opposition, good idea, moving to `src/Controllers`.

File: src/Controllers/FilesController.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// files.js
```
Comment (gfosco) [inline src/Controllers/FilesController.js]: Comment no longer true. ;)

File: src/Controllers/FilesController.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+var bodyParser = require('body-parser'),
```
Comment (maysale01) [inline src/Controllers/FilesController.js]: Seems pedantic, but you can replace these with ES6 imports!

File: src/Controllers/FilesController.js (hunk 1 above)
Comment (nlutsenko) [inline src/Controllers/FilesController.js]: Nice catch.

**Comment by nlutsenko** [comment]
Updated with ES6 import style and updated the comment.

**Pull request merged**